### PR TITLE
Bugfix/unr 5126 nfr fix

### PR DESCRIPTION
**Pull request by improbable-danny**
#### Description
WellKnownEntitySystem is not initialized sometimes when this callback is triggered (via NetDriver::InitCoreClasses)

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp [hunk 1]
```diff
@@ -865,9 +865,12 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 		return;
 	}
 
-	if (IsServer() && WellKnownEntitySystem.IsValid())
+	if (IsServer())
 	{
-		WellKnownEntitySystem->OnMapLoaded();
+		if (WellKnownEntitySystem != nullptr)
```
Comment (m-samiec) [inline SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp:870]: This is the same as an IsValid check

Comment (improbable-danny) [reply SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp:870]: Yeah, it changed the else behavior which I think assumed clients

Comment (improbable-danny) [inline SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp:870]:
```suggestion
		if (WellKnownEntitySystem.IsValid())
```

File: SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp [hunk 2]
```diff
@@ -865,9 +865,12 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 		return;
 	}
 
-	if (IsServer() && WellKnownEntitySystem.IsValid())
+	if (IsServer())
 	{
-		WellKnownEntitySystem->OnMapLoaded();
+		if (WellKnownEntitySystem.IsValid())
```
Comment (improbable-danny) [inline SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp:870]: The main thing here is IsServer doesn't imply WellKnownEntitySystem != null (routing worker is both a server and has an invalid WellKnownEntitySystem)

**Review by m-samiec — APPROVED** [review]

**Review by MatthewSandfordImprobable — APPROVED** [review]

**Pull request merged**